diff --git a/dist/mongoserver.js b/dist/mongoserver.js
index 7d8a59e7ce26f3f8832a939338af4ec532fa49f2..df8a3e1e86ffd957d2bdb20b332342b243cc31b8 100644
--- a/dist/mongoserver.js
+++ b/dist/mongoserver.js
@@ -282,7 +282,7 @@ class MongoServer extends events_1.EventEmitter {
             if (this.childProcess.exitCode === null &&
                 this.childProcess.signalCode === null) {
                 this.childProcess.kill('SIGKILL');
-                await (0, events_1.once)(this.childProcess, 'exit');
+                await (0, events_1.once)(this.childProcess, 'close');
             }
             (0, util_1.debug)('stopped running process');
             this.childProcess = undefined;
